### ISSUE TITLE
TeaVM IDEA plugin broken after update to 0.9.0-dev-1

### ISSUE THREAD

**Issue by Ihromant** [konsoletyper/teavm#679]
Error running ${configurationName}
Could not start daemon. Stderr: Error: Could not find or load main class org.teavm.idea.devserver.DevServerRunner Caused by: java.lang.ClassNotFoundException: org.teavm.idea.devserver.DevServerRunner
[image](https://10255720379752917787.googlegroups.com/attach/192aac515e703/Screenshot_2023-03-26_22-25-07.png?part=0.1&view=1&vt=ANaJVrEELc121Qw66pWsdHjUP8Lcruihh5WQo4r-p50a0MPPhfZ7cNG7bdmBXtuwS8kmZPHvGh_WVEQ0WU00i7Z4SAMvFT1rCDuwfDxN2vSNdc6sE6rYQdw)

**Comment by Ihromant** [comment]
If fix is long to make - how to revert to 0.8.0-dev-2?

**Comment by Ihromant** [comment]
Seems that no version of IDEA plugin coulnd't be installed. Tried to install 0.8.0 from global repository - same result. I noticed strange name of plugin jar in IDEA plugins folder:
Is in normal name for plugin? I suppose that it causes error. Also in IDEAIC2022.3 folder there is no plugin jar (but there are other custom plugin jars).
[image](https://10255720379752917787.googlegroups.com/attach/1954ff93dabaf/Screenshot_2023-03-26_23-13-15.png?part=0.1&view=1&vt=ANaJVrFtXk0ejYNmSaEucH5n-rGJUmdglvlHatd56845E7bmvGkKgPi6-n6GYdA1tItul_ERbyV6Uf_kKvsXEQgsz5yRQMKi8fhIz1ug5JvKv2a5EcSVqYU)
